@plotly/dash-component-plugins/src/dynamicImport.js

```diff
@@ -0,0 +1,31 @@
+import { lazy } from 'react';
+
+export const decorate = (target, promise) => {
```

Comment:
do you want to just call this `asyncDecorator` if that's how it'll be used?

Reply:
sure

Reply:
https://github.com/plotly/dash/pull/899/commits/af3500637e1403dc2719654118c835e72212c4b4